README.md

```diff
@@ -83,14 +83,26 @@ False
 >>>
 ```
 
-To use with ASUS Tinker Board:
+### To use with ASUS Tinker Board:
+In `def __init__()` in file `__init__.py`:
+```
+board_type: BoardType = BoardType.TINKER_BOARD
+```
+In `def _create_argument_parser()` in file `cli.py`:
+```
+default="tinker-board"
+```
 
-```python
->>> from rpi_backlight import Backlight, BoardType
->>>
->>> backlight = Backlight(board_type=BoardType.TINKER_BOARD)
->>> # continue like above
+### To use with ASUS Tinker Board 2:
+In `def __init__()` in file `__init__.py`:
+```
+board_type: BoardType = BoardType.TINKER_BOARD_2
+```
+In `def _create_argument_parser()` in file `cli.py`:
 ```
+default="tinker-board-2"
+```
+
```

Comment:
Hmm, that's not really what I meant (and doesn't seem like an improvement over the current example. More like this:

````
```python
>>> from rpi_backlight import Backlight, BoardType
>>>
>>> backlight = Backlight(board_type=BoardType.TINKER_BOARD)
>>> # continue like above
```

When using the ASUS Tinker Board 2, set `board_type` to `BoardType.TINKER_BOARD_2`.
````

Reply:
How do we prompt the user to also make the changes in the second file too, cli?
As there was a change needed in both files for it to work.
I tried changing it in only one place, and it was still thinking I was using a Raspberry Pi.

Reply:
OK I made the changes as requested.
In my fork, once you accept the pull, I'll change the default in both places
Is there any way to have it so the default only needs to be set in one place?

Reply:
OK I made the changes as requested.